spec/tasks/assets_tasks_spec.rb

```diff
@@ -0,0 +1,40 @@
+require "rails_helper"
+
+RSpec.describe "asset tasks" do
+  Rails.application.load_tasks
+
+  describe "update_topical_event_legacy_url_paths" do
+    let(:task) { Rake::Task["assets:update_topical_event_legacy_url_paths"] }
+
+    def legacy_url(asset_name, model_name)
+      "/government/uploads/system/uploads/#{model_name}/file/1234/#{asset_name}.jpeg"
+    end
+
+    def assert_assets_the_same_except_legacy_url_path(actual_assets, expected_assets)
+      ignored_keys = %w[legacy_url_path updated_at last_modified created_at] # annoyingly there's a really small difference in created at which is I think due to mongo truncating
```

Comment:
Now I think about it, I am a bit worried about the difference in created at here, maybe there is an issue?